_data/changelogs/component-header.yml

```
@@ -2,6 +2,12 @@ title: Header
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Added Header variants table to guidance page.
```

We should not capitalize the component name.

```suggestion
    summary: Added header variants table to guidance page.
```